plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -91,6 +91,8 @@ func ClusterRoles() []rbac.ClusterRole {
 			Rules: []rbac.PolicyRule{
 				// TODO add future selfsubjectrulesreview, project request APIs, project listing APIs
 				rbac.NewRule("create").Groups(authorizationGroup).Resources("selfsubjectaccessreviews").RuleOrDie(),
+
+				rbac.NewRule(Read...).Groups(storageGroup).Resources("storageclasses").RuleOrDie(),
```

Comment:
Do either of my proposed names resonate, either as good ideas or as negatives to push toward the right idea? If I had a name, I could revise the proposal to use a separate role.